bigquery/main.go

```diff
@@ -26,11 +26,28 @@ import (
 	"cloud.google.com/go/civil"
 	"github.com/GoogleCloudPlatform/cloud-build-notifiers/lib/notifiers"
 	log "github.com/golang/glog"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/google/go-containerregistry/pkg/name"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/google"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
 	cbpb "google.golang.org/genproto/googleapis/devtools/cloudbuild/v1"
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
 var tableResource = regexp.MustCompile(".*/.*/.*/(.*)/.*/(.*)")
 
+var terminalStatusCodes = map[cbpb.Build_Status]bool{
+	cbpb.Build_SUCCESS:        true,
+	cbpb.Build_FAILURE:        true,
+	cbpb.Build_INTERNAL_ERROR: true,
+	cbpb.Build_TIMEOUT:        true,
+	cbpb.Build_CANCELLED:      true,
+	cbpb.Build_EXPIRED:        true,
+}
+
+const megaByte = int64(1000000)
```

Comment:
Minor paranoia (totally fine to leave as a `// TODO(aricz): ` or `// TODO(ljr): `) but we should confirm that the size is in MB and not [MiB](https://en.wikipedia.org/wiki/Mebibyte).